msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -150,7 +150,7 @@ This option validates the zipped app package with validation rules.
 
 :::image type="content" source="../assets/images/teams-toolkit-v2/customize app manifest/validate-app-package.png" alt-text="Screenshot shows the selection of Validate app package using validation rules.":::
 
-Alternatively, use the following Teams Toolkit CLI command:
+Alternatively, use the following Agents Toolkit CLI command:
 
 ```bash
 teamsfx validate --app-package-file-path <YOUR-PATH-TO-APP-PACKAGE>
```

Comment:
ALL cli command should be update to "atk"

Reply:
@v-preethah  - Please update all the files (if applicable)